src-frontend/app/controllers/procedure.js

```diff
@@ -19,9 +19,20 @@ export default Ember.Controller.extend({
         return pageModels;
     },
 
+    generateErrorString: function(errors, type) {
+        var message = 'Something went wrong creating the ' + type;
+        errors.forEach(function(error){
+            for (var attr in error.details) {
+                message = attr.toString() + ": " + error.details[attr][0];
```

Comment:
No, cause attr will be the attribute such as `display_index`, so we access the value of that message like if it was `display_index: "Can't be blank!"`